exonum/src/blockchain/schema.rs

```diff
@@ -62,6 +62,9 @@ pub struct Schema<T> {
     view: T,
 }
 
+/// Constants for the 'ord' values used in prefix generation
+const CONSENSUS_MESSAGES_CACHE_ORD: u8 = 9;
```

Comment:
:+1: 

How about adding other constants?

Reply:
It was my initial intention ;)